mono/tests/finalizer-exception.cs

```diff
@@ -31,7 +31,7 @@ public class FinalizerException {
 			Environment.Exit (0);
 		};
 
-		var t = new Thread (delegate () { MakeException (1024); });
+		var t = new Thread (delegate () { MakeException (128); });
```

Comment:
this is suspicious. did the frame size of the interpreter main loop grow?

Reply:
@lewurm EXCEPTION_CHECKPOINT contains a local var exc, and with O0 gcc doesn't bother to reuse the same stack slot for it.

Reply:
there are about 70 branch instructions

Reply:
that does make sense. on CI we do `-O2` though, so I feel like we should keep the old value to catch "real" frame size bumps.

Reply:
I made the change so I don't get bothered with this test when compiling with O0 locally, but I see your point